src/main/java/org/datadog/jmxfetch/AppConfig.java

```diff
@@ -204,6 +204,13 @@
     @Builder.Default
     private int ipcPort = 0;
 
+    /**
+     * Boolean setting to determine whether to ignore jvm_direct instances.
+     * If set to true, other instances will be ignored.
+     */
+    @Builder.Default
+    private boolean allowDirectInstances = false;
```

Comment:
The "allow" in the name doesn't convey that `If set to true, other instances will be ignored.`.

Maybe something like `targetDirectInstances`?

Reply:
Yeah, I was having a hard time coming up with a better name... I like that better though.  Thanks for the suggestion.

Reply:
Done.